test/commands/version.test.ts

```diff
@@ -19,22 +22,19 @@ describe('version', () => {
   .stdout()
   .command(['version', '--verbose'])
   .end('runs version --verbose', output => {
-    expect(output.stdout).to.equal(` CLI Version:
-\t@oclif/plugin-version/${pjson.version}
-
- Architecture:
-\t${process.platform}-${process.arch}
-
- Node Version:
-\tnode-${process.version}
-
- Plugin Version:
-\t
-
- OS and Version:
-\t${osType()} ${osRelease()}
-
-`)
+    expect(output.stdout).to.contain(` CLI Version:
+\t@oclif/plugin-version/${pjson.version}`)
+    expect(output.stdout).to.contain(` Architecture:
+\t${process.platform}-${process.arch}`)
+    expect(output.stdout).to.contain(` Node Version:
+\tnode-${process.version}`)
+    expect(output.stdout).to.contain(' Plugin Version:')
+    expect(output.stdout).to.contain(` OS and Version:
+\t${osType()} ${osRelease()}`)
+    expect(output.stdout).to.contain(` Shell:
+\t${getShell()}`)
+    expect(output.stdout).to.contain(` Root Path:
+\t${process.cwd()}`)
```

Comment:
I'd prefer to use `equal` here instead of `to.contain` because it's more restrictive, therefore ensuring the output is exactly what we expect and nothing more.